llvm/lib/CodeGen/AsmPrinter/AccelTable.cpp

```diff
@@ -559,37 +587,61 @@ void llvm::emitDWARF5AccelTable(
     default:
       continue;
     }
-    CUIndex[CU.index()] = Count++;
+    CUIndex[CU.index()] = CUCount++;
     assert(CU.index() == CU.value()->getUniqueID());
     const DwarfCompileUnit *MainCU =
         DD.useSplitDwarf() ? CU.value()->getSkeleton() : CU.value().get();
     CompUnits.push_back(MainCU->getLabelBegin());
   }
 
+  for (const auto &TU : enumerate(TUs)) {
```

Comment:
Why is this using `enumerate` if it's only using the values anyway? Perhaps this could iterate over `TUs` directly instead?

Reply:
I was following the example of CUs iteration to try to keep coding style consistent.

Reply:
This isn't a stylistic thing in this case - the CU iteration needed indexes, so it used `enumerate`, whereas the TU iteration doesn't need them, so it shouldn't use `enumerate`, I think.

(but, again, happy to wait on resolving a bunch of this stuff until we better understand the general approach/memory concerns, etc)